code/modules/admin/admin_verbs.dm

```diff
@@ -52,6 +52,7 @@ var/list/admin_verbs_admin = list(
 	/client/proc/manage_silicon_laws,	// Allows viewing and editing silicon laws. ,
 	/client/proc/check_antagonists,
 	/client/proc/admin_memo,			//admin memo system. show/delete/write. +SERVER needed to delete admin memos of others,
+	/client/proc/mentor_memo,			//Ditto but for mentors
```

Comment:
Append , to the comments in lists please, that should solve the check not clearing.

Reply:
I don't even know how or why it worked when it is a **COMMENT** but it did. Thanks.

Global var hash is still screaming at me which I'm not sure how to fix.